packages/engine-server/src/migrations/migrations.ts

```diff
@@ -39,6 +42,21 @@ export const ALL_MIGRATIONS: Migrations[] = [
       },
     ],
   },
+  {
+    version: "0.51.4",
+    changes: [
+      {
+        name: "don't switch to legacy preview if not currently on it",
```

Comment:
Hmm I'm a little confused with the logic here. It looks like we're **not** converting existing users who haven't tried out preview v2 yet to start using preview v2?

Reply:
yeah, the plan is to try out preview v2 with new users and have it bake for a bit before migrating existing users